src/components/Select/Select.tsx

```diff
@@ -25,28 +31,44 @@ export const Select: FunctionComponent<SelectProps> = ({
 	options,
 	id,
 	disabled = false,
+	loading = false,
+	clearable = false,
+	placeholder,
 	value,
 	onChange = () => {},
 }) => {
-	function onValueChange(event: ChangeEvent<HTMLSelectElement>) {
-		onChange(event.target.value);
+	function onValueChange(changedValue: ValueType<SelectOption>, actionMeta: ActionMeta) {
+		if (!value) {
+			return;
+		}
+		if (actionMeta.action !== 'create-option') {
+			onChange((changedValue as SelectOption).value);
+		}
 	}
 
+	const styleFunctions = {
```

Comment:
Kan je ook gwn targetten met css dacht ik via `uw-custom-class__option--is-selected`

Reply:
nice, ik heb ernaar gezocht en vond het nergens in de docs, en ik kan die dropdown menu niet inspecteren want het verdwijnt steeds als hij focus verliest